mm/2s2h/Enhancements/Songs/SkipScarecrowSong.cpp

```diff
@@ -21,7 +22,13 @@ void RegisterSkipScarecrowSong() {
          */
         if ((enKakasi->picto.actor.xzDistToPlayer < enKakasi->songSummonDist) &&
             ((BREG(1) != 0) || (gPlayState->msgCtx.ocarinaMode == OCARINA_MODE_ACTIVE))) {
+            // In Rando we may utilize Ocarina Buttons, ensure this is honored.
+            if (IS_RANDO && Rando::Logic::canPlaySong(OCARINA_SONG_SCARECROW_SPAWN)) {
```

Comment:
Oops, my suggestion did the opposite and made Pierre spawn if you don't have enough notes, and he won't spawn if you do. Minor fix:

```suggestion
            if (IS_RANDO && !Rando::Logic::canPlaySong(OCARINA_SONG_SCARECROW_SPAWN)) {
```